src/events/message/index.ts

```diff
@@ -29,13 +30,15 @@ export default class MessageEvent extends Event {
 				message.content.trim().toLowerCase(),
 			)
 		) {
-			const pingMessage = await message.channel.send("Ping ?").catch(noop);
+			const pingMessage = await message.channel
+				.send(`Ping ? *Cela peut durer jusqu'à ${REQUEST_TIMEOUT / 1000}s.*`)
```

Comment:
```suggestion
				.send(`Ping ? *Cela peut durer jusqu'à ${REQUEST_TIMEOUT / 1000} secondes.*`)
```
(excuse moi mais c'est important !)